standards/11-clipboard.md

```diff
@@ -0,0 +1,41 @@
+# *COS 11:* Clipboard
+
+## Quick information
+| Information |                                                                                        |
+| ----------- | -------------------------------------------------------------------------------------- |
+| Version     | 1.0.0                                                                                  |
+| Type        | Protocol                                                                               |
+
+### Technical Details
+Clipboard defines a method to share information like text with other programs.
+
+It is a Table called "clipboard" in the Shell Environment.
+
+#### The Clipboard Table
+It is a Table called "clipboard" in the Shell Environment.
+
+| Header      | Description                                                                            |
+| ----------- | -------------------------------------------------------------------------------------- |
+| type        | A String, that Describes, what is in the Clipboard (see at Point "Types")              |
```

Comment:
Describes isn't even a noun... Don't capitalise anything unless it's at the beginning of a sentence or a proper noun (names of people or places, etc.).